.github/workflows/integration_tests.yaml

```
@@ -0,0 +1,41 @@
+name: XCUI Tests
+on:
+  workflow_dispatch:
+    inputs:
+      SDK_TARGET_VERSION:
+        description: "Target a specific SDK version. Otherwise latest"
+        required: false
+  push:
+
+jobs:
+  build:
```

Personally, I'd prefer having this explicit. Unless we add a description. If we ever split this into separate jobs however, the explicit name can help indicate to someone that it should be. 

```suggestion
  build-and-test
```